SPECS/python-urllib3/python-urllib3.spec

```diff
@@ -1,14 +1,15 @@
 Summary:        A powerful, sanity-friendly HTTP client for Python.
 Name:           python-urllib3
 Version:        1.26.18
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Languages/Python
 URL:            https://pypi.python.org/pypi/urllib3
 Source0:        https://github.com/urllib3/urllib3/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
 BuildArch:      noarch
+Patch0: urllib3_test_recent_date.patch
```

Comment:
Please align spacing with the other tags above.

Reply:
Good call. Thanks.